OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -328,7 +340,7 @@ protected virtual void FireBarrel(Actor self, IFacing facing, in Target target,
 					if (projectile != null)
 						self.World.Add(projectile);
 
-					if (args.Weapon.Report != null && args.Weapon.Report.Length > 0)
+					if (playSound && args.Weapon.Report != null && args.Weapon.Report.Length > 0)
```

Comment:
Just checking if `playSound` is also supposed to guard the burst sound? If it's currently correct then using a more specific name since it plays some sounds and not other would be nice.

Reply:
To be fair I can just remove this. Are sound engine wrapper already takes care of sound spam anyway